cli/internal/runsummary/format_execution_summary.go

```diff
@@ -26,14 +32,36 @@ func (rsm *Meta) printExecutionSummary() {
 		}
 	}
 
-	if summary.ExecutionSummary.attempted == 0 {
+	if attempted == 0 {
 		ui.Output("") // Clear the line
 		ui.Warn("No tasks were executed as part of this run.")
 	}
 
-	ui.Output("") // Clear the line
-	ui.Output(util.Sprintf("${BOLD} Tasks:${BOLD_GREEN}    %v successful${RESET}${GRAY}, %v total${RESET}", summary.ExecutionSummary.cached+summary.ExecutionSummary.success, summary.ExecutionSummary.attempted))
-	ui.Output(util.Sprintf("${BOLD}Cached:    %v cached${RESET}${GRAY}, %v total${RESET}", summary.ExecutionSummary.cached, summary.ExecutionSummary.attempted))
-	ui.Output(util.Sprintf("${BOLD}  Time:    %v${RESET} %v${RESET}", time.Since(summary.ExecutionSummary.startedAt).Truncate(time.Millisecond), maybeFullTurbo))
+	ui.Output("")    // Clear the line
+	spacer := "    " // 4 chars
+
+	// We'll start with some default lines
+	lines := []string{
+		util.Sprintf("${BOLD} Tasks:${BOLD_GREEN}%s%v successful${RESET}${GRAY}, %v total${RESET}", spacer, success, attempted),
+		util.Sprintf("${BOLD}Cached:%s%v cached${RESET}${GRAY}, %v total${RESET}", spacer, cached, attempted),
+		util.Sprintf("${BOLD}  Time:%s%v${RESET} %v${RESET}", spacer, duration, maybeFullTurbo),
+	}
+
+	// If we have a run summary file, add that in. We have to do some whitespace adjusting, so just
+	// duplicate the lines above instead of using some fancy tabbibg/spacing mechanism.
+	if rsm.getPath().FileExists() {
+		lines = []string{
+			util.Sprintf("${BOLD}  Tasks:${BOLD_GREEN}%s%v successful${RESET}${GRAY}, %v total${RESET}", spacer, success, attempted),
+			util.Sprintf("${BOLD} Cached:%s%v cached${RESET}${GRAY}, %v total${RESET}", spacer, cached, attempted),
+			util.Sprintf("${BOLD}   Time:%s%v${RESET} %v${RESET}", spacer, duration, maybeFullTurbo),
+			util.Sprintf("${BOLD}Summary:%s%s${RESET}", spacer, rsm.getPath()),
```

Comment:
note that this is an absolute path. I considered using a relative path, but it could be confusing if running from within a subdirectory of a monorepo (with global turbo), so this makes more sense